lib/rubocop/cop/ezcater/migration_documentation.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Ezcater
+      # Enforce use of migration best practices by adding comment of documentation url
+      #
+      # @example
+      #   # bad
+      #   # frozen_string_literal: true
+      #
+      #   class CreateUsers < ActiveRecord::Migration[5.1]
+      #     def change
+      #
+      #   # good
+      #   # frozen_string_literal: true
+      #   # https://ezcater.atlassian.net/wiki/spaces/POL/pages/53248012/Adding+a+migration
```

Comment:
will this conflict with existing cops? i think we require a newline after the magic frozen string comment

Reply:
ohhh good to know. well that's fun

Reply:
let me investigate and make changes if needed